osu.Game/Graphics/ParticleSpewer.cs

```diff
@@ -171,7 +178,7 @@ protected override void Blit(IRenderer renderer)
                         transformPosition(rect.BottomRight, rect.Centre, angle)
                     );
 
-                    renderer.DrawQuad(Texture, quad, DrawColourInfo.Colour.MultiplyAlpha(alpha),
+                    renderer.DrawQuad(Texture, quad, p.Colour.MultiplyAlpha(alpha),
```

Comment:
Not sure this is correct? This means that the spewer's colour is discarded entirely and the only colour that actually matters is the particle's?

Reply:
at least it is so in the stable version. If we use spewer's color, there is a serious bug where all the particles' color changes when we want to switch colors for new particles.